config/environments/test.rb

```diff
@@ -120,7 +120,9 @@
   ###################################################################
   # Location of the public and private key for the git user on the system
   GITOLITE_SETTINGS = { public_key: "/home/git/git.pub",
-                        private_key: "/home/git/.ssh/id_rsa" }
+                        private_key: "/home/git/.ssh/id_rsa",
+			host: "localhost" }
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.<br>Tab detected.